The default upstream configuration expects mdb to be available by default: as
we build it as a module, ensure it's loaded.

Author: Daniel Watkins <daniel.watkins@chainguard.dev>
--
diff --git a/servers/slapd/slapd.conf b/servers/slapd/slapd.conf
index a868793..89c9bd2 100644
--- a/servers/slapd/slapd.conf
+++ b/servers/slapd/slapd.conf
@@ -15,7 +18,7 @@ argsfile	%LOCALSTATEDIR%/run/slapd.args
 
 # Load dynamic backend modules:
 # modulepath	%MODULEDIR%
-# moduleload	back_mdb.la
+moduleload	back_mdb.so
 # moduleload	back_ldap.la
 
 # Sample security restrictions
